src/App.js

```diff
@@ -1,13 +1,12 @@
 import React from 'react'
 
+import AppLayout from './core/components/layout'
+
+import 'antd/dist/antd.css'
```

Comment:
You can import `import 'antd/dist/antd.css'`  in /App.scss as  `@import '~antd/dist/antd.css'; i think is better